lib/nameable.rb

```diff
@@ -0,0 +1,5 @@
+module Nameable
+  def correct_name
+    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+  end
+end
```

Comment:
![image](https://github.com/Grandi0z/school_library/assets/93039359/e1183ea7-065e-480d-9f3f-e196770036aa)

- [ ] It would be nice if you make this Nameable a class instead of module to make your app follow the decorator pattern and project requirements 👍